### ISSUE TITLE
Low reproduction performance of the baseline_large

### ISSUE THREAD

**Issue by YuanxianH** [OpenDriveLab/OpenLane-V2#39]
Hi, I trained the `baseline_large` in 4 V100 with the default config you guys provided (without change). However, the reproduction performance is much lower than the val benchmark. The metrics we obtain are as follows:
- DET_l: 7.8543
- DET_t: 43.6408
- TOP_ll: 0.0001
- TOP_lt: 4.22
- OLS: 18.21

Is this normal? I noticed the performance you mentioned in another [issue](https://github.com/OpenDriveLab/OpenLane-V2/issues/22#issuecomment-1511048980) is much better than ours. Are they the same model? Would you mind providing the training log of the `baseline_large` or the one using InternImage as backbone?

**Comment by faikit** [comment]
By default the model is trained in 8 gpus. When adjusting the batch size, pls also adjust the learning rate accordingly.

**Comment by qwertyczx** [comment]
Hello, I just trained the baseline_large in 8 A100 with the default config too. I got results as follows:

- DET_l: 0.1106
- DET_t: 0.4947
- TOP_ll: 0.0002
- TOP_lt: 0.0533

The metrics are also lower than your results on the val split. What could be the possible reasons for this?

**Comment by faikit** [comment]
Hi, your results look good. The configs and settings in the reported result in #22 are a little bit different to the provided one.
